packages/core/lib/services/dids/config.js

```diff
@@ -0,0 +1,25 @@
+import assert from 'assert';
+
+export const serviceName = 'didManager';
+export const validation = {
+  keypairToDidKeyDocument: ({keypairDoc}) => {
+    assert(typeof keypairDoc === 'object', 'invalid keypairDoc');
+    assert(keypairDoc.type === 'Ed25519VerificationKey2018', 'invalid type');
+
+    assert(
+      typeof keypairDoc.publicKeyBase58 === 'string',
+      'publicKeyBase58 is not present',
+    );
+  },
+  getDidResolution: ({didDocument}) => {
+    assert(typeof didDocument.id === 'string', 'invalid didDocument');
+  },
+};
+
+export type KeypairToDidKeyDocumentParams = {
+  keypairDoc: any,
+};
+
+export type GetDidResolutionParams = {
```

Comment:
nitpick I noticed in Ossy's PR too. We seem to be inconsistent with naming of functions, variables etc. Sometimes we use `DID`, sometimes `Did` and sometimes `DiD`. We should pick one.

Reply:
I would go with `DID`. I can update that

Reply:
I've pushed a refactor, would be nice to @osinakayah review too. To make sure thats ok. It will require an update in the app as well. I will push another PR in the app repository